roles/cockpit/templates/cockpit.conf.j2

```diff
@@ -0,0 +1,5 @@
+[WebService]
+Origins = https://{{ cockpit_web_url }}
```

Comment:
If you hadn't removed https from the variable it could have been plugged directly in.